pkg/reconciler/labeler/config/store.go

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Should this re-use the one from above?

Reply:
you mean should reuse the configuration store?

Reply:
yeah is there a reason to have individual ones here?
It is fine, but I want it to be an informed decision.

Reply:
so the issue #8871 specified to have a separate store for each @whaught
After a second look at the code I find it makes more sense to reuse knative.dev/serving/pkg/reconciler/route/config though

Reply:
updated labeler to use configuration store, but I have my doubts regarding separation of concerns (coming from a java background) I know they will point to the same configmaps at the end, but